modules/python/compat/compat-python-v3.c

```diff
@@ -21,9 +21,23 @@
  *
  */
 
+#include "compat-python.h"
+
 void
 py_init_argv(void)
 {
   static wchar_t *argv[] = {L"syslog-ng"};
   PySys_SetArgvEx(1, argv, 0);
 }
+
+PyObject *
+py_as_int(gint num)
+{
+  return PyLong_FromLong(num);
+};
+
+PyObject *
+slng_PyMethod_New(PyObject *func, PyObject *class)
+{
```

Comment:
I would rather use a prefix the is not related to syslog-ng, e.g. compat_PyMethod_New or something along those lines, that clearly indicates that it is only a wrapper around PyMethod_New, and exists for compatibility reasons only.